features/step_definitions/general_steps.rb

```diff
@@ -2,16 +2,16 @@
 
   @app = App.new
   @app.front_office_home_page.load
-  @app.front_office_home_page.start_button.click
+  # @app.front_office_home_page.start_button.click
 
 end
 
 Given(/^I have a valid username and password$/) do
 
   # Back office login page
   @app.login_page.submit(
```

Comment:
Can you update the configuration section of the readme please? you can probably just copy from here:
https://github.com/DEFRA/waste-carriers-acceptance-tests#configuration